ingest/src/main/java/com/connexta/ingest/transform/TransformRequest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.transform;
+
+import java.io.Serializable;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class TransformRequest implements Serializable {
```

Comment:
✏️Use a constructor and final fields instead of setters so that we can enforce that the values are never null and shouldn't be changed